assets/src/edit-story/components/colorPicker/useReduction.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Modified from https://github.com/anater/useReduction
+ */
```

Comment:
I couldn't use the original, as it imported directly from `react` and thus clashed with _something_.

Reply:
I'll merge from master and see if that fixes it. But tomorrow.

Reply:
Hmm that library doesn't specify any license 😕

Reply:
Ah, MIT it seems: https://github.com/anater/useReduction/blob/a0b5887ffd2966c67c239ad0f5232d59e609a4d1/package.json#L16

All good.

Reply:
Yes, I did check that. Seems compatible.

Reply:
I've now imported the `use-reduction` package from npm, and it seems to work fine.

Reply:
And no it doesn't. We'll keep using the same forked version in stead - it's pretty compact anyway, but moving it to general utils, as it's nice for situations where you need a smaller reducer.